core/app/[locale]/(default)/webpages/contact/[id]/_actions/submit-contact-form.ts

```diff
@@ -0,0 +1,119 @@
+'use server';
```

Comment:
🍹 Any reason we moved this action outside of `/contact-us`?

Reply:
@jorgemoya Just following the same pattern we use for the `register` route. It felt a bit redundant / unnecessary to have a `/contact-us` directory when we're already in the `/contact/[id]` directory. So after cleaning up the old implementation, this would have been the only file in there (`/contact-us/_actions/submit-contact-form.ts`, so I figured it made more sense to just move `_actions` under the `/contact/[id]` route 👍